src/app/pages/login/login.page.html

```diff
@@ -1,49 +1,51 @@
 <ion-header>
-  <ion-toolbar color="dark">
-    <ion-title>DES Ionic 5 - Login Screen</ion-title>
-  </ion-toolbar>
+    <ion-toolbar mode="ios">
+        <ion-title>Login Screen</ion-title>
```

Comment:
on DES 3 the title is `Driving Examiner Services`

Reply:
changed